### ISSUE TITLE
civitai-helper integration?

### ISSUE THREAD

**Issue by tranzmatt** [alexandersokol/sd-model-organizer#37]
The civitai-helper extension auto generates a json .info file when you download a model from civit ai.  Being able to scan for those files and parse the info needed would a wonderful thing.

**Comment by alexandersokol** [comment]
In case you have already downloaded model with civitai-helper and .info file (json data) this file will be visible as local file in model organizer, and when you presses "Add" button, a new record fields will be prefilled from this file.

**Comment by alexandersokol** [comment]
Yes, redownloading isn't an option, thats why it should scan for local files. Do you have any errors in the console while pressing `Reload` button in model organizer tab?

**Comment by alexandersokol** [comment]
Looks kinda interesting are using custom directories arguments or simlinks?

**Comment by alexandersokol** [comment]
If your model file has .info file with same name it should take all possible data from it. Could you share one of your .info files that doesn't have it?

I also can see that you have dark theme but fields are white but should be grey, feels like something wrong with theme detection, are you using `?__theme=dark` address argument or you are starting webui with `--theme dark`?

**Comment by alexandersokol** [comment]
Thanks! I found a bug - I really skipped "description" field to import, I will fix it as soon as possible. But from other hand .info file contains only short description ` "description": "<p>More details added<br />Blue saturation improved<br />All colors have been balanced</p>",` and seems this is not what are you looking for...

I planned new feature that allows to prefill model record by its hash from civitai, but unfotunely currently I don't have enough time even to start it. I guess It might be usefull in case like yours. 

And I finnaly figured out why model organizer uses incorect color theme - seems you are using dark theme in Windows, and by default it applies to webui without any paramenters, I need to investigate how to deal with it too.

**Comment by Mange** [comment]
In addition, having the ability to re-import the same information from Civitai on an already existing record (essentially "refresh metadata") could give users a way to migrate away from the civitai-helper extension and into this one.

First import the JSON data like we do currently, then refresh the metadata from Civitai on the record to then populate the thumbnails, descriptions, etc.
